packages/util-fns/obj_util/index.test.js

```diff
@@ -0,0 +1,24 @@
+const { mapObj } = require('.');
+
+describe('mapObj', () => {
+  it('never changes an empty object', () => {
```

Comment:
if you have `should` in rest of the cases, might as well use it here.

Reply:
Will change